<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_job</class>
 <widget class="QWidget" name="change_job">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="position_name_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="position_name_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>96</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Назва посади</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_position_button">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>230</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Змінити посаду</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="is_teacher_checkbox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>150</y>
     <width>99</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Викладач</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>130</y>
     <width>231</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="subject_label">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Предмет</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="subject_picker">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="new_job_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Змінити посаду</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="pick_position_combobox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
